.github/workflows/rust.yml

```diff
@@ -419,3 +419,38 @@ jobs:
 
       - if: "failure()"
         uses: "andymckay/cancel-action@271cfbfa11ca9222f7be99a47e8f929574549e0a" # v0.4
+
+  no-std-tests:
+    name: "Test (no_std)"
+    runs-on: ubuntu-latest
+    needs: [machete, docs]
```

Comment:
I'd incline to start this test after the following `needs: [clippy, wasm_clippy, check, wasm_check, docs]`

Reply:
I don't agree, this no-std check/test should be cheap and runs faster than for example `clippy` it's not really a test

Reply:
In that case, we'd need to adjust the `needs: [clippy, wasm_clippy, check, wasm_check, docs]` to include `no-std-tests` as well :D

Reply:
ok, sounds good :)